examples/stable_diffusion_v2/tools/safety_checker/nsfw_model.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+
+import mindspore as ms
+from mindspore import Parameter, Tensor, nn, ops
+
+
+class Normalization(nn.Cell):
+    def __init__(self, shape):
+        super().__init__()
+        self.mean = Parameter(Tensor(np.zeros(shape), ms.float32))
+        self.variance = Parameter(Tensor(np.ones(shape), ms.float32))
```

Comment:
PyTorch's `register_buffer` stores non-trainable states. So, you can set `requires_grad=False` in `Parameter` or use `Tensor`.

Reply:
fixed